pages/wi/session/voting/review.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { NextSeo } from 'next-seo';
+import { useRouter } from 'next/router';
+import debug from 'debug';
+import ContentSection from '../../../../components/shared/ContentSection';
+import User from '../../../../components/User';
+import { useFetchUser } from '../../../../hooks/user';
+import Header from '../../../../components/Session/Voting/Review/Header';
+import Content from '../../../../components/Session/Voting/Review/Content';
+
+const _ = require('lodash');
+
+const dlog = debug('that:session:create');
+
+const SessionVoting = () => {
+  dlog('session voting');
+
+  const router = useRouter();
+  const { user, loading: loadingUser } = useFetchUser();
```

Comment:
Page already handles this for you, you just need to destructure them from props.